### PR TITLE
fixed spelling and grammatical errors

### PR DESCRIPTION
**Pull request by ShubhGurukul**
fixed spelling and grammatical errors

[DOCS-XXX](https://algorithmia.atlassian.net/browse/DOCS-XXX) - Brief description / title of ticket

Additional context/notes if appropriate (e.g., specifics about what's changing, links to epic or other PRs, etc.)

## Checklist
- [ ] My PR title includes a relevant Jira ticket name
- [ ] If no associated Jira ticket, my PR title is informative (individual commit messages are squashed in this repo)
- [ ] Unnecessary text (notes, TODOs, etc.) has been removed
- [ ] For YAML "front matter":
  - Properties are alphabetized
  - `permalink` and `redirect_from/to` properties have `/` at beginning and end (e.g., `/path/`)
- [ ] Screenshots, if present, follow the [Screenshot guidelines](https://algorithmia.atlassian.net/wiki/spaces/CUSTOMERS/pages/1634861478/CFD+Style+Guide#Screenshots) which means:
  - Full browser width (browser at 100% zoom and 10-12" width)
  - Max 1000px
- [ ] Grammar, style, tone, punctuation, etc. follow the Algorithmia [manual of style](https://docs.google.com/document/d/1PPVfgMkX7-EVGLPMhN1E485CAXu9QfhSLKM0lZhnZdU/edit?usp=sharing)
- [ ] Headers are “Sentence case with Proper Nouns capitalized”
- [ ] Placeholder strings are `UPPER_SNAKE_CASE` and are named as in the [Glossary](https://algorithmia.com/developers/glossary) where appropriate

### CODE REVIEW

File: _pages/platform/organization-profile.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ tags: [basics]
 title: "Organizations"
 ---
 
-If you're looking to share algorithms privately or publish algorithms on behalf of an organization, such as a university, private company, or open-source community, then you should consider using our organizations feature.
+If you're looking to share algorithms privately or publish algorithms on behalf of an organization, such as a university, private company, or open-source community, then you should consider using our organization's feature.
```
Comment (lemonez) [inline _pages/platform/organization-profile.md]: This is actually already correct as is; "organizations" is what we call the feature, so it's plural and not possessive.

Comment (lemonez) [inline _pages/platform/organization-profile.md]:
```suggestion
If you're looking to share algorithms privately or publish algorithms on behalf of an organization, such as a university, private company, or open-source community, then you should consider using our organizations feature.
```

File: _pages/platform/organization-profile.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ The following sections cover the basics of using organizations on the Algorithmi
 
 ## Profile page contents
 
-The profile overview provides a rich source of information about an organization. In the overview, you can find the name of an organization, its members, and a list of the algorithms under its ownershipt. If you're an administrator for the organization, you'll also be able to invite and delete members, as well as edit basic information about the organization.
+The profile overview provides a rich source of information about an organization. In the overview, you can find the name of an organization, its members, and a list of the algorithms under its ownership. If you're an administrator for the organization, you'll also be able to invite and delete members, as well as edit basic information about the organization.
```
Comment (lemonez) [inline _pages/platform/organization-profile.md:29]: Thanks!

File: _pages/platform/organization-profile.md [hunk 3]
```diff
@@ -50,7 +50,7 @@ Once your organization has been created you can edit all the provided informatio
 
 ### Inviting users to an organization
 
-You can invite users to your organization by clicking the **Invite Member** button at the upper-right corner of the organization profile. You can invite members either by using their Algorithmia account name or by providing their email address. Users invited by account name will receive an email and as well as an in-platform notification to join the organization.
+You can invite users to your organization by clicking the **Invite Member** button at the upper-right corner of the organization profile. You can invite members either by using their Algorithmia account name or by providing their email addresses. Users invited by account name will receive an email and as well as an in-platform notification to join the organization.
```
Comment (lemonez) [inline _pages/platform/organization-profile.md]:
```suggestion
You can invite users to your organization by clicking the **Invite Member** button at the upper-right corner of the organization profile. You can invite a member either by using their Algorithmia account name or by providing their email address. Users invited by account name will receive an email and as well as an in-platform notification to join the organization.
```
Thanks; the use of "their" was intended to be singular and gender neutral, but I see it wasn't clear. I've updated the wording to be clearer, how does this sound?

File: _pages/platform/organization-profile.md [hunk 4]
```diff
@@ -71,12 +71,12 @@ Once members of your organization have created an algorithm and submitted it for
 
 **Note:** Every new version of the algorithm to be published will require independent approval.
 
-At this time you'll be able to set the royalty (if any) if you're using our non-Enterprise platform, and finalize publishing.
+At this time you'll be able to set the royalty (if any) if you're using our non-Enterprise platform and finalize publishing.
```
Comment (lemonez) [inline _pages/platform/organization-profile.md:74]: Thanks; everything after this point actually applies to our deprecated public marketplace platform and will be going away, but I can make those changes myself.

**Review by lemonez — CHANGES_REQUESTED** [review]
Thanks for your contribution!

**Review by ToshMeston — DISMISSED** [review]
👍 🚢

**Pull request merged**